src/main/scala/Player.scala

```diff
@@ -0,0 +1,6 @@
+case class Player(
+                   name: String,
+                   rack: List[Tile],
+                   wordsPlayed: List[String],
+                   gameOver: Boolean
```

Comment:
If we were playing scrabble would we normally think about a Player's game being over, or the whole game being over? Does it make sense to model this at the `Player` level?